Libplanet.Tests/Blocks/BlockTest.cs

```diff
@@ -201,11 +201,11 @@ public void EvaluateActions()
                 .EvaluateActionsPerTx(address => null)
                 .ToImmutableArray();
             int randomValue = 0;
-            (int, int, string[])[] expectations =
+            (int, int, string[], Address)[] expectations =
             {
-                (0, 0, new[] { "A", null, null, null, null }),
-                (0, 1, new[] { "A", "B", null, null, null }),
-                (1, 0, new[] { "A", "B", "C", null, null }),
+                (0, 0, new[] { "A", null, null, null, null }, _fx.TxFixture.PrivateKey1.PublicKey.ToAddress()),
+                (0, 1, new[] { "A", "B", null, null, null }, _fx.TxFixture.PrivateKey1.PublicKey.ToAddress()),
```

Comment:
This can be shortened like:

```suggestion
                (0, 1, new[] { "A", "B", null, null, null }, _fx.TxFixture.Address1),
```

Reply:
Fixed. (cf5eedbe757b312358635dec47e565ae0b46e671)